sycl/test-e2e/KernelCompiler/sycl_device_flags.cpp

```diff
@@ -6,7 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-// REQUIRES: level_zero
+// REQUIRES: (level_zero || cuda || amd)
```

Comment:
How does this test work on cuda/hip? IIRC it greps for some string in the IGC shader dumps.